File: Python/selection_sort.py
def selectionSort(array, size):
    for i in range(size):
        min_idx = i
        for j in range(i+1, size):
            if array[j] < array[min_idx]:
                min_idx = j
        array[i], array[min_idx] = array[min_idx], array[i]


data = [-2, 45, 0, 11, -9]
size = len(data)
selectionSort(data, size)
print("Sorted array in ascending order : ", data)

File: Data Structure/Selection_sort.cpp
#include<iostream>
#include<bits/stdc++.h>
using namespace std;


int Selection_sort(int arr[],int len);
int main()
{
  int len;
  cout<<"Enter the size of array : ";
  cin>>len;
  int arr[len];
  cout<<"Enter the Element of array : ";
  for(int i=0;i<len;i++)
  {
    cin>>arr[i];
  }
  Selection_sort(arr,len);
 return 0;
}


int Selection_sort(int arr[],int len)
{
    int min,temp,j,i;
    for( i=0;i<len-1;i++)
    {
        min=i;
        for( j=i+1;j<len;j++)
        {
            if(arr[j]<arr[min])
            {
                min=j;
            }
        }
        temp=arr[i];
        arr[i]=arr[min];
        arr[min]=temp;
    }
    cout<<"Sorted array is accending order : ";
    for(int i=0;i<len;i++)
    {
        cout<<arr[i]<<" ";
    }
    cout<<endl;

    cout<<"Sorted array is descending order : ";
    for(int i=len-1;i>=0;i--)
    {
        cout<<arr[i]<<" ";
    }
    
}